### PR TITLE
add license, contributing.md, and update readme

### PR DESCRIPTION
**Pull request by coffee-cup**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,21 +4,31 @@
...
 
 - [Node via NPM](https://www.npmjs.com/)
 - [Node via Yarn](https://yarnpkg.com/)
 - [Go](https://golang.org)
 - [Rust](https://www.rust-lang.org/)
 - [Deno](https://deno.land)
 
-More languages will be added very soon
+- ✨ **Intutive defaults**: In most cases, building and deploying and app with nixpacks should _just work_ with no configuration needed.
+- ⚙️ **Customization where necessary**: Everypart of the pipeline should be customizable. These include the [Nix packages](https://search.nixos.org/packages) to add to the environment and build/start commands.
```
Comment (gschier) [inline README.md:20]: Every part

File: README.md [hunk 2]
```diff
@@ -4,21 +4,31 @@
...
-Nixpacks currently supports
+You can follow along with the roadmap in the [GitHub project](https://github.com/railwayapp/nixpacks/projects/1).
+
+## Core Ideas
 
 - [Node via NPM](https://www.npmjs.com/)
 - [Node via Yarn](https://yarnpkg.com/)
 - [Go](https://golang.org)
 - [Rust](https://www.rust-lang.org/)
 - [Deno](https://deno.land)
```
Comment (gschier) [inline README.md:15]: These aren't core ideas are they?

**Review by gschier — APPROVED** [review]
Love it. Just a few minor comments

File: README.md (hunk 2 above)
Comment (coffee-cup) [reply README.md:15]: Oh oops, merge conflict 😅

**Pull request merged**